requirements/requirements.txt

```diff
@@ -9,6 +9,7 @@ jax
 jaxlib
 jaxmarl
 jumanji @ git+https://github.com/sash-a/jumanji
+lbforaging @ git+https://github.com/Louay-Ben-nessir/lb-foraging.git
```

Comment:
Just a general question, why are we depending on forks for the environments?

Reply:
The original implementation stalls when using newer versions of gym https://github.com/semitable/lb-foraging/issues/20